examples/00-mapdl-examples/bracket_static.py

```diff
@@ -0,0 +1,743 @@
+
+
+Problem specification
+=====================
+
++---------------------------+---------------------------------------------------------------------------------------------------------+
+| Applicable Products:      | Ansys Multiphysics, Ansys Mechanical, Ansys Structural                                                  |
++---------------------------+---------------------------------------------------------------------------------------------------------+
+| Level of Difficulty:      | Easy                                                                                                    |
++---------------------------+---------------------------------------------------------------------------------------------------------+
+| Interactive Time Required:| 60 to 90 minutes                                                                                        |
++---------------------------+---------------------------------------------------------------------------------------------------------+
+| Discipline:               | Structural                                                                                              |
++---------------------------+---------------------------------------------------------------------------------------------------------+
+| Analysis Type:            | Linear static                                                                                           |
++---------------------------+---------------------------------------------------------------------------------------------------------+
+| Element Types Used:       | `PLANE183 <elem_plane183_>`_                                                                            |
++---------------------------+---------------------------------------------------------------------------------------------------------+
+| Features Demonstrated:    | Solid modeling including primitives, boolean operations, and fillets; tapered pressure load             |
+|                           | deformed shape and stress displays; listing of reaction forces;                                         |
++---------------------------+---------------------------------------------------------------------------------------------------------+
+| Help Resources:           | Structural Static Analysis and `PLANE183 <elem_plane183_>`_                                             |
++---------------------------+---------------------------------------------------------------------------------------------------------+
+
+
+Problem description
+===================
+
+This is a simple, single-load-step, structural static analysis of a corner angle
+bracket. The upper-left pin hole is constrained (welded) around its entire
+circumference, and a tapered pressure load is applied to the bottom of the
+lower-right pin hole. The US Customary system of units is used.
+The objective is to demonstrate a typical Mechanical APDL analysis procedure.
```

Comment:
```suggestion
The objective is to demonstrate how Mechanical APDL is typical used in an analysis.
```
Is there an effort underway to always use Mechanical APDL (rather than MAPDL) when referring to the Ansys product?

Reply:
I always get confused with terms... 

Pinging @koubaa

Reply:
> "Mechanical APDL" in prose.

I guess we should use "Mechanical APDL" in text wherever possible. :)